eng/Packages/BuildOnly.props

```diff
@@ -9,13 +9,15 @@
     <PackageVersion Include="Grpc.Tools" Version="2.45.0" />
     <PackageVersion Include="LibGit2Sharp" Version="0.27.0-preview-0175" />
     <PackageVersion Include="Microsoft.Build.Framework" Version="17.4.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore" Version="4.4.0-6.23101.13" />
```

Comment:
The inbox .NET 8 ASP.NET Core analysers (i.e., packs\Microsoft.AspNetCore.App.Ref\8.0.0-preview.5.23260.10\analyzers\dotnet\cs) depend on Microsoft.CodeAnalysis.ExternalAccess.AspNetCore, Version=4.4.0.0. However, this assembly is nowhere to be found publicly. So I had to reference the closest version available on https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-tools/ feed.
This prompted the version updates for other Microsoft.CodeAnalysis.* libraries.

I raised the issue in https://teams.microsoft.com/l/message/19:ab93d6b5dae043a2bef1fb0be802b685@thread.skype/1686021636857?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=4ba7372f-2799-4677-89f0-7a1aaea3706c&parentMessageId=1686021636857&teamName=.NET%20Developer%20Experience&channelName=Analyzers%20and%20Source%20Generators&createdTime=1686021636857.

Reply:
Should we add a comment above these lines that links to an issue that tracks this so we remove once it is no longer needed?